AstroCal/control/control.py

```diff
@@ -206,3 +206,64 @@ def getMoonStatus():
 
     moon_percent_rounded = round(moon_percent)
     return result, "Illumination " + str(moon_percent_rounded) + "%"
+    
+    
+
```

Comment:
Not seeing any changes to console_output.py. That means there is no way for me to see this in the console at the moment. Please add an extra option in the moon_menu in console_output.py. 
![image](https://user-images.githubusercontent.com/58247240/193940723-1e6bb411-1872-4871-b212-e578df06fd6d.png)